examples/platform/qpg/app/main.cpp

```diff
@@ -120,14 +125,19 @@ void Application_Init(void)
     ChipLogProgress(NotSpecified, "Qorvo " APP_NAME " Launching");
     ChipLogProgress(NotSpecified, "============================");
 
-    error = GetAppTask().Init();
+#if defined(BASE_APP_BUILD)
```

Comment:
we generally use `#if` and never `#ifdef` or equivalent (like `#if defined` here). That makes it easier to spot intentional changes vs 'forgot to set something'

Reply:
Prefer #if defined() for maintainability. If we change the define name we usually look for all instances and change it overall.

Reply:
defined will cause relevant includes to be missed. It is important to not use `ifdef` or `if defined`